packages/babel-traverse/src/path/modification.ts

```diff
@@ -338,8 +373,9 @@ export function pushContainer(
   const path = NodePath.get({
     parentPath: this,
     parent: this.node,
-    container: container,
+    container: container as unknown as t.Node | t.Node[],
```

Comment:
I have to cast it as unknown first because `NodePath.get` does not have template typings.